### PR TITLE
Add St Brigids Day and Easter Monday Holiday for Ireland

### PR DESCRIPTION
**Pull request by tobya**
I have added code for the new St Brigids day in Ireland.

This is a new holiday from 2023 onwards and is First Monday of February unless Feb1 falls on a friday.

I have added tests and all seems to be in order.

Happy to make any futher changes I need to.

I also added a small section on adding tests when a contributor adds code.  I hope this is ok.

### CODE REVIEW

File: src/Provider/Ireland/Ireland.php [hunk 1]
```diff
@@ -87,6 +93,23 @@ private function getOctoberHoliday(int $year, int $additionalType = HolidayType:
...
     }
 
+    /**
+     * St Brigids day is a new holiday (2023) that is celebrated on first monday of February unless Feb 1 falls on
+     * a friday then it is on that Friday.
+     */
+    private function getStBrigidsDayHoliday(int $year, int $additionalType = HolidayType::OTHER): Holiday
+    {
+        $dateToCheck = new DateTime("{$year}-02-01");
+        $dateInfo = getdate($dateToCheck->getTimestamp());
```
[the remark this answers is not in the document]
Comment (tobya) [reply src/Provider/Ireland/Ireland.php]: I  have now changed this to be more similar to other code.

File: README.md [hunk 2]
```diff
@@ -438,6 +438,20 @@ Contributions are highly welcome. Please follow these rules when submitting a PR
 
 By submitting a PR you agree that all your contributed code may be used under the MIT license.
 
+### Adding Tests 
+
+    - Provide tests inside `tests/Provider/...` that ideally cover all edge cases. This can be done as follows:
```
[the remark this answers is not in the document]
Comment (tobya) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -438,6 +438,20 @@ Contributions are highly welcome. Please follow these rules when submitting a PR
 
 By submitting a PR you agree that all your contributed code may be used under the MIT license.
 
+### Adding Tests
```
[the remark this answers is not in the document]
Comment (tobya) [reply README.md:441]: I removed the duplication

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- St Brigids Day Holiday for Ireland
```
[the remark this answers is not in the document]
Comment (tobya) [reply CHANGELOG.md:11]: I have added this translation

**Comment by tobya** [comment]
@umulmrum Thank you, I have made those changes now.

**Comment by umulmrum** [comment]
Thank you @tobya!
